azurerm/internal/services/cdn/cdn_profile_data_source.go

```diff
@@ -37,10 +41,20 @@ func dataSourceArmCdnProfile() *schema.Resource {
 
 			"tags": tags.SchemaDataSource(),
 		},
+
+		SchemaVersion: 1,
+		StateUpgraders: []schema.StateUpgrader{
+			{
+				Type:    resourceArmCdnProfile().CoreConfigSchema().ImpliedType(),
+				Upgrade: migration.CdnProfileV0ToV1,
+				Version: 0,
+			},
```

Comment:
this is a Data Source so it doesn't need a migration (this'll all be updated every time) - so we can remove this